prime-router/src/main/kotlin/azure/Event.kt

```diff
@@ -195,17 +169,11 @@ class ProcessEvent(
 ) : Event(eventAction, at) {
     override fun toQueueMessage(): String {
         // turn the defaults and route to into strings that can go on the process queue
```

Comment:
We should remove this comment, its not really accurate anymore. After your update, it's obvious we are serializing the params via QueueMessage which is a common task and shouldn't need a comment anymore.